src/DotNetWorker/Http/HttpResponseDataExtensions.cs

```diff
@@ -63,6 +66,92 @@ public static Task WriteStringAsync(this HttpResponseData response, string value
             return response.Body.WriteAsync(bytes, 0, bytes.Length);
         }
 
+        /// <summary>
+        /// Write the specified value as JSON to the response body using the provided <see cref="ObjectSerializer"/>.
+        /// The response content-type will be set to <code>application/json; charset=utf-8</code> and the status code set to 200.
+        /// </summary>
+        /// <typeparam name="T">The type of object to write.</typeparam>
+        /// <param name="response">The response to write JSON to.</param>
+        /// <param name="instance">The instance to serialize and write as JSON.</param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> used to cancel the operation.</param>
+        /// <returns>A <see cref="ValueTask"/> that represents the asynchronous operation.</returns>
+        public static ValueTask WriteAsJsonAsync<T>(this HttpResponseData response, T instance, CancellationToken cancellationToken = default)
+        {
+            return WriteAsJsonAsync(response, instance, "application/json; charset=utf-8", cancellationToken);
+        }
+
+        /// <summary>
+        /// Write the specified value as JSON to the response body using the default <see cref="ObjectSerializer"/> configured for this worker.
+        /// The response content-type will be set to <code>application/json; charset=utf-8</code> and the status code set to 200.
```

Comment:
This content-type note doesn't seem right here (since it can be set in a param).

Reply:
Copy and paste error :(